src/main/java/org/isf/patient/gui/PatientInsertExtended.java

```diff
@@ -2111,8 +2111,11 @@ private JPanel getJRightPanel() {
 			jRightPanel = new JPanel(new BorderLayout());
 
 			try {
-				final Image image = patient.getPatientProfilePhoto() != null ? patient.getPatientProfilePhoto().getPhotoAsImage() : null;
-				photoPanel = new PatientPhotoPanel(this, patient.getCode(), image);
+				PatientProfilePhoto photo = this.patientManager.retrievePatientProfilePhoto(patient);
+				final Image image = photo != null ? photo.getPhotoAsImage() : null;
+				Image scaledImage = image != null ? ImageUtil. scaleImage(image, PatientGuiConst.IMAGE_THUMBNAIL_MAX_WIDTH) : null;
```

Comment:
Why a space in the middle of `ImageUtil. scaleImage()` ?

Reply:
Removed, thanks!